internal/controller/policyserver_controller.go

```diff
@@ -80,12 +80,12 @@ func (r *PolicyServerReconciler) Reconcile(ctx context.Context, req ctrl.Request
 		return r.reconcileDeletion(ctx, &policyServer, policies)
 	}
 
-	policyServerCARootSecret, err := r.fetchOrInitializePolicyServerCARootSecret(ctx, &policyServer)
+	CARootSecret, err := r.reconcileInternalCARootSecret(ctx, &policyServer)
```

Comment:
unfortunately, we should use `caRootSecret` here since Pascal-cased local (unexported) variables are not idiomatic in go